PHYSOR26/common_mesh.i

```diff
@@ -0,0 +1,126 @@
+ref_mesh_file_name=gt.e-s011
+test_mesh_file_name=hex_same_flux_high_err_ma.e-s010
+
+[Mesh]
+  [generated_mesh]
+     type = GeneratedMeshGenerator
+    nx = 20
+    ny = 4
+    nz = 4
+    xmin = 0.0
+    xmax = 100.0
+    ymin = 0.0
+    ymax = 10.0
+    zmin = 0.0
+    zmax = 10.0
+    dim = 3
+  []
+[]
+
+
+[AuxVariables]
```

Comment:
I think it would make most sense to have these in this order:
* ref_solution_hierarchy
* test_solution_hierarchy
* max_element_hierarchy
* adaptive_hierarchy
* element_refinement_step